src/app/siteMapPage/page.tsx

```diff
@@ -18,26 +17,56 @@ const SiteMap = dynamic(
     loading: () => <p>Loading...</p>,
   },
 );
+type ModeState = 'tours' | 'exhibits';
+
+
 
 /**
  * @returns Page for the interactive map
  */
 function MapPage() {
+
+  const [selectedMap, setSelectedMap] = useState(filterButtonContent[0]); // "Virtual Tour Map" by default
+  const [mode, setMode] = useState<ModeState>('tours');
+
+  // move tour logic here: need to share state between filter 
+  const handleFilter = (mapName) => {
+    setSelectedMap(mapName);
+    if (mapName === "Virtual Tour Map") {
+      setMode("tours");
+    } else if (mapName === "Exhibits Map") {
+      setMode("exhibits");
+    }
+  };
+
+  const renderFilterContainer = () => (
+    <div className="container mb-6  pl-0 pr-0 ">
```

Comment:
container class is making weird max-width interactions. also this should have a w-full i think